kratos/mpi/tests/sources/test_flag_synchronization.cpp

```diff
@@ -145,10 +145,10 @@ KRATOS_DISTRIBUTED_TEST_CASE_IN_SUITE(MPIDataCommunicatorFlagsAndOperations, Kra
     Kratos::Flags flags;
     //       both true | both false | opposite sets | first true   | first false | second true | second false
     if (world_rank == root) {
-        flags = ACTIVE | NOT_RIGID  | STRUCTURE     | MPI_BOUNDARY | NOT_PERIODIC;
+        flags = ACTIVE | RIGID.AsFalse()  | STRUCTURE    | MPI_BOUNDARY | PERIODIC.AsFalse();
```

Comment:
did you run those tests also in MPI?

Reply:
No. In any case these changes do not affect the parallelism.

Reply:
Hm I wouldn't be so sure, there are functions to sync the flags, which if I remember correctly somehow use the flags and its internals
can you please run the tests?

Reply:
I've just run the MPI suite. Everything works as expected :wink:

Reply:
Ok thx
No worries, I am anyway planning to soon add the MPI tests to the CI so you will be blocked from merging if you screw things up :)))